recipes/ccphylo/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "ccphylo" %}
+{% set version = "0.6.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://bitbucket.org/genomicepidemiology/{{ name }}/get/{{ version }}.tar.gz
+  sha256: d23860f90df6d546f340d53f294070070406b4f80e7e1f3d8ecbfaaefb7ca9bf
+
+build:
+  skip: True  # [win]
+  number: 0
+
+requirements:
+  build:
+    - make
+    - {{ compiler('c') }}
+  host:
+    - zlib
+  run:
+    - zlib
+  
+test:
+  commands:
+   - ccphylo --help | grep CCPhylo >/dev/null
+
+about:
+  home: https://bitbucket.org/genomicepidemiology/ccphylo
+  summary: 'CCPhylo is a software suite designed to easen phylogenitic 
+analysis based on alignments from KMA. Neighbor-Joining can be run 
+without KMA alignments, if a Phylip distance matrix is given using 
+ccphylo tree.'
+  license: Apache-2.0
+  license_file: LICENSE.txt
+  doc_url: doi:10.1093/biomethods/bpab008
```

Comment:
I don't think this will render in a good way. I believe this needs to be a proper URL. (Or do we support DOIs here by now?)